pycbc/workflow/jobsetup.py

```diff
@@ -1166,19 +1356,92 @@ class LalappsInspinjExecutable(Executable):
     """
     The class used to create jobs for the lalapps_inspinj Executable.
     """
-    current_retention_level = Executable.FINAL_RESULT
-    def create_node(self, segment):
+    current_retention_level = Executable.CRITICAL
+    def __init__(self, cp, exe_name, universe=None, ifos=None, out_dir=None,
+                 tags=[]):
+        Executable.__init__(self, cp, exe_name, universe, ifos, out_dir,
+                            tags=tags)
+        self.cp = cp
+        self.tags = tags
+
+    def create_node(self, segment, exttrig_file=None, tags=[]):
         node = Node(self)
 
-        if self.get_opt('write-compress') is not None:
-            ext = '.xml.gz'
```

Comment:
Removing this is dangerous, as giving this option would then provide incorrect output? Maybe just fail if coh_PTF cannot run with this option and coh_PTF is being used.